src/Event/Recommendations/DBALRecommendationRepository.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\Recommendations;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\FetchMode;
+
+final class DBALRecommendationRepository implements RecommendationsRepository
```

Comment:
There's a naming discrepancy here between `Recommendation` and `Recommendations`. I suggest we choose the singular version for the interface as well.

Reply:
Nicely spotted! I went for a consistent plural naming because recommendations are always a collection.